sklearn/svm/_classes.py

```diff
@@ -1550,6 +1605,9 @@ class OneClassSVM(OutlierMixin, BaseLibSVM):
     n_support_ : ndarray of shape (n_classes,), dtype=int32
         Number of support vectors for each class.
 
+        .. deprecated::
+            `n_support_` was deprecated in version 1.1 and will be removed in 1.3.
```

Comment:
I think `n_support_` still makes sense to have in `SVR` and `OneClassSVM` because it is the number of support vectors. In this case, the `n_support_`'s docstring can be updated to say:

> n_support_ : ndarray of shape (1,), dtype=int32

What do you think @amueller?